io-engine/src/grpc/v1/snapshot.rs

```diff
@@ -157,14 +157,14 @@ impl From<VolumeSnapshotDescriptor> for SnapshotInfo {
         Self {
             snapshot_uuid: s.snapshot_lvol().uuid(),
             snapshot_name: s.snapshot_params().name().unwrap_or_default(),
-            snapshot_size: s.snapshot_lvol().size(),
+            snapshot_size: s.snapshot_lvol().usage().allocated_bytes,
             num_clones: s.num_clones(),
             timestamp: s
                 .snapshot_params()
                 .create_time()
                 .map(|s| s.parse::<DateTime<Utc>>().unwrap_or_default().into()),
             source_uuid: s.source_uuid(),
-            source_size: s.source_size(),
+            source_size: s.snapshot_lvol().size(),
```

Comment:
`s.snapshot_lvol().size()` will return what size ? How it is different from replica lvol size ?
If it is not same, Do We need to add another parameter in gRPC to send back replica lvol size after snapshot is taken ?

Reply:
Snapshot and replica have the same size (not allocated space, but 'size', i.e. maximum allocated space).